### PR TITLE
Update week 1 2 3

### PR DESCRIPTION
**Pull request by saripovdenis**

### CODE REVIEW

File: modules/js-fe-developer/README.md [hunk 1]
```diff
@@ -31,13 +31,13 @@ Understanding NodeJS, Cloud services, and TypeScript is crucial due to the expan
 - 🔍 **NodeJS** allows developers to understand back-end functionalities and contribute comprehensively to project development.
 - 🚀 **Cloud Services** proficiency ensures the optimal utilization of modern development resources, emphasizing performance and scalability.
 
-## 🔗 Useful Links
+## 🔗 Education materials
```
Comment (Anik188) [inline modules/js-fe-developer/README.md]: Educational Materials

File: modules/rs-school-intro/README.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ The course documentation answers most queries. However, it's worth noting that t
     - Culture of communication
     - Different roles within the community
 
-## Theory
+## Education materials
```
Comment (Anik188) [inline modules/rs-school-intro/README.md]: Educational Materials

Comment (saripovdenis) [reply modules/rs-school-intro/README.md]: fixed

File: modules/js-fe-developer/README.md (hunk 1 above)
Comment (saripovdenis) [reply modules/js-fe-developer/README.md]: fixed

**Review by Anik188 — APPROVED** [review]

**Pull request merged**